infra/modules/azure_monorepo_single_env_starter_pack/github_environments_cd.tf

```diff
@@ -0,0 +1,100 @@
+resource "github_repository_environment" "infra_prod_cd" {
+  environment = "infra-prod-cd"
+  repository  = github_repository.this.name
+
+  deployment_branch_policy {
+    protected_branches     = false
+    custom_branch_policies = true
+  }
+
+  reviewers {
+    teams = matchkeys(
+      data.github_organization_teams.all.teams[*].id,
+      data.github_organization_teams.all.teams[*].slug,
+      local.infra_cd.reviewers_teams
+    )
+  }
```

Comment:
```suggestion
# Add deployment protection rules
  deployment_protection_rules {
    required_reviewers      = true
    prevent_self_review    = true
    required_conversations = false
  }

  }
```

Reply:
Where did you get this property? I can't [see it](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_environment)